fuzz/fuzz_targets/fuzz_skrifa_outline.rs

```diff
@@ -0,0 +1,127 @@
+#![no_main]
+use std::{error::Error, fmt::Display};
+
+use libfuzzer_sys::{
+    arbitrary::{self, Arbitrary, Unstructured},
+    fuzz_target,
+};
+use skrifa::{
+    instance::Size,
+    outline::{DrawError, DrawSettings, HintingInstance, HintingMode, OutlinePen},
+    raw::tables::glyf::ToPathStyle,
+    FontRef, MetadataProvider,
+};
+
+/// The pen for when you don't really care what gets drawn
+struct NopPen;
```

Comment:
This might be at some point useful to have in Skrifa itself. I have a NopPen in Skia for some cases as well.